Resources/ServerInfo/_NF/Guidebook/Rules/13_Command.xml

```diff
@@ -22,4 +25,11 @@ You are the highest authority in the sector apart from Central Command. You must
     -- Granting docking extensions.
     -- Restricting vendor access (e.g. placing machines behind locked doors).
     -- Restricting access to station resources.
+    - The Sheriff or SR are under no obligation to offer to purchase contraband from other players. Seizure of illegal goods is always allowed under Space Law.
```

Comment:
Could add something about "must act in good faith" and that they cannot use bait and switch tactics on buybacks

Reply:
If the NFSD have a reason to believe someone has been, say, farming contraband for the purpose of selling to the NFSD, then they are well within their rights to prosecute that person. NFSD should _never_ be under any kind of obligation to pay for contraband, and should _always_ be free to prosecute as they see fit.

Reply:
Sometimes contraband deals turn out to not be in good faith specifically because evidence arises that suggests the person has been using or abusing those items, or they've been used for crime and therefore seizure is the better option.

Reply:
Yeah there's extremes in both cases, but sheriff's shouldn't be allowed to do a deliberate bait and switch either. 
How could something be worded to prevent the intentional bait and switch while still allowing leeway to prosecute mercs acting in bad faith? 
What separates "farming contraband to sell" from "getting it off the streets and turning it in to NFSD"?

Reply:
What defines a "deliberate bait and switch" is a matter of interpretation and very definitely depends on which side you're on and how entitled the merc feels to guns they're not allowed (within Space Law) to possess. In my mind, this is a good example of a natural IC conflict that can lead to good roleplay. As a mercenary, you should _never_ feel comfortable owning and using C3 contraband, and should _always_ worry about it potentially being taken away. I, for one, don't want rules that allow mercs or NFSD to rule lawyer their way around playing their respective roles.

Reply:
> What defines a "deliberate bait and switch" is a matter of interpretation and very definitely depends on which side you're on and how entitled the merc feels to guns they're not allowed (within Space Law) to possess. In my mind, this is a good example of a natural IC conflict that can lead to good roleplay. As a mercenary, you should _never_ feel comfortable owning and using C3 contraband, and should _always_ worry about it potentially being taken away. I, for one, don't want rules that allow mercs or NFSD to rule lawyer their way around playing their respective roles.

I think this one is pretty easy. Deliberate would clearly be, the sheriff advertises buybacks just to lure mercs in to arrest or confiscate without payment. The sheriff never had the intent to pay the advertised reward.

Reply:
Personally I think that's a great example of events leading to reasonable escalation and player conflict. 
Police forces do operate hand-in schemes and sometimes run sting operations in order to deal with people who have knowingly used illegal items and tried to then take advantage of such things. I don't see a problem with this. IC issue.